internal/pkg/aws/ecs/service.go

```diff
@@ -70,6 +70,44 @@ func (s *Service) ServiceStatus() ServiceStatus {
 	}
 }
 
+// ServiceConnectAliases returns the ECS Service Connect client aliases for a service.
+func (s *Service) ServiceConnectAliases() []string {
+	if len(s.Deployments) == 0 {
+		return nil
+	}
+	lastDeployment := s.Deployments[0]
+	scConfig := lastDeployment.ServiceConnectConfiguration
+	if scConfig == nil || !aws.BoolValue(scConfig.Enabled) {
+		return nil
+	}
+	ns := aws.StringValue(scConfig.Namespace)
+	var aliases []string
+	for _, service := range scConfig.Services {
+		defaultName := aws.StringValue(service.PortName)
+		if aws.StringValue(service.DiscoveryName) != "" {
+			defaultName = aws.StringValue(service.DiscoveryName)
+		}
+		defaultAlias := fmt.Sprintf("%s.%s", defaultName, ns)
```

Comment:
nit
```suggestion
		defaultAlias := fmt.Sprintf("%s.%s", defaultName, aws.StringValue(scConfig.Namespace)
```
(or even just `scConfig.Namespace`, let go do the deref)

Reply:
good catch!